doc/contributing.rst

```diff
@@ -158,5 +158,81 @@ Happy testing!
 Connecting new machine learning libraries
 =========================================
 
-Coming soon - please stay tuned!
-
+Hosting the library
+~~~~~~~~~~~~~~~~~~~
+
+Each extension created should be a stand-alone repository, compatible with the
+`OpenML repository <https://github.com/openml/openml-python>`_.
+The extension repository should work off-the-shelf with openml-python installed.
+
+Create a `public Github repo <https://help.github.com/en/articles/create-a-repo>`_ with
+the following directory structure:
+
+::
+
+| [repo name]
+|   |-- extensions
```

Comment:
We don't need this directory level. In fact, it might rather been hurtful as the user then cannot install the extension as a package if it is called `extensions`.

Reply:
In that case, should I include a setup.py in the directory tree, to indicate or reinforce that we encourage them to publish as a separate package?